projects/plugins/jetpack/modules/masterbar/inline-help/class-inline-help.php

```diff
@@ -32,7 +32,11 @@ public function register_actions() {
 		// Do not inject the FAB icon on embedded screens since the parent window may already contain a FAB icon.
 		$is_framed = ! empty( $_GET['frame-nonce'] );
 
-		if ( $is_framed ) {
+		// Do not inject the FAB icon on Yoast screens to avoid overlap with the Yoast help icon.
+		$current_page = \filter_input( \INPUT_GET, 'page', \FILTER_SANITIZE_STRING );
```

Comment:
We may want to look at [`get_current_screen`](https://developer.wordpress.org/reference/functions/get_current_screen/) as an alternative to this, but the information isn't available until [the `current_screen` hook](https://developer.wordpress.org/reference/hooks/current_screen/), so we would probably need to have this method called on that hook, by changing the constructor in this class. Something like:

```php
add_action( 'current_screen', array( $this, 'register_actions' ) );
```

That action passes in the current screen object too, so we could change this method to accept that parameter and avoid calling `get_current_screen`.

It looks like the screen ID always starts `seo_page_wpseo_` for Yoast admin pages, but that would still pass the `strpos` check here.

Reply:
Nice thanks! 🎉  I changed the method to accept the parameter: https://github.com/Automattic/jetpack/pull/19890/commits/834c49760d9998b2622db9b1481834a4f5af64e4

The top level Yoast admin page screen ID starts with `toplevel_page_wpseo_`, so I kept `wpseo`.